kubernetes/values.yaml

```diff
@@ -1,4 +1,4 @@
-localnodes: 3
+localnodes: 2
```

Comment:
Why are u decreasing the localnodes?
Is there a limitation?

Reply:
This is the default value, could be changed to something more. For local deployments no more is needed and in CI there is no disk space for more.